LayoutKit/Internal/CGSizeExtension.swift

```diff
@@ -10,12 +10,16 @@ import CoreGraphics
 import Foundation
 
 extension CGSize {
-    func decreasedByInsets(_ insets: EdgeInsets) -> CGSize {
-        return CGSize(width: width - insets.left - insets.right, height: height - insets.top - insets.bottom)
+    func increased(by insets: EdgeInsets, lineFragmentPadding: CGFloat = 0) -> CGSize {
```

Comment:
The handling of `lineFragmentPadding` seems text-specific and shouldn't be in such a general CGSize extension.